README.md

```diff
@@ -1,81 +1,141 @@
-<div align="center">
+<br />
+<p align="center">
   <img src="https://user-images.githubusercontent.com/1626923/137092657-fb398d20-b592-4661-a1f9-4135db0b61d5.png" alt="Vue Storefront" height="80px" />
```

Comment:
Maybe out of the PR's context, but it would be great to replace this image with one with 80px height to be responsible for mobile devices. I've opened the README.md in the GitHub App, and the image stretches.

![image](https://user-images.githubusercontent.com/9027363/156771192-1974787d-f8b8-47b6-84df-157e270a50bd.png)

Reply:
Fixed!